.ci/bump-stack-version.sh

```diff
@@ -24,6 +24,25 @@ else
 	SED="sed -i"
 fi
 
+echo "Update stack with version ${VERSION} in .stack-version"
+echo "${VERSION}" > .stack-version
+git add .stack-version
+
+echo "Update stack with version ${VERSION} in Go files"
+find . -name 'defaults.go' -path './internal/common/*' -print0 |
```

Comment:
There is only one file:

![image](https://user-images.githubusercontent.com/2871786/126619907-86606891-a111-4b8c-be10-8436e05cbc0c.png)

Maybe a simple sed should be enough?

Reply:
Done in https://github.com/elastic/e2e-testing/pull/1351/commits/fe83a86b26f9e5a055985cabcd29423b6a0ba292